includes/plugins/woocommerce-subscriptions/class-subscriptions-meta.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * WooCommerce Subscriptions meta class.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Main class.
+ */
+class Subscriptions_Meta {
+	const CANCELLATION_REASON_META_KEY        = 'newspack_subscriptions_cancellation_reason';
+	const CANCELLATION_REASON_USER_CANCELLED  = 'user-cancelled';
+	const CANCELLATION_REASON_ADMIN_CANCELLED = 'manually-cancelled';
```

Comment:
Every time I see this value I have to ask myself if manually means by the admin or by the user. Why not call this admin-cancelled?